app/admin/governance/multitenant_wallet/wallet_admin.py

```diff
@@ -10,6 +10,7 @@
 from fastapi import APIRouter, Depends, HTTPException
 from pydantic import BaseModel
 
+# from app.config import get_settings
```

Comment:
?

Reply:
?... not sure what happened there. sorry